gulpfile.babel.js

```diff
@@ -122,7 +122,7 @@ const buildConfig = (invocationArgs, sourceRoot, testRoot, buildRoot) => {
   config.files.src.js = `${config.dir.src.js}/**/*.js`;
   config.files.src.jsEntryPoint = `${config.dir.src.js}/${invocationOptions.jsEntryPoint}`;
   config.files.src.images = `${config.dir.src.images}/**/*`;
-  config.files.src.favicon = `${config.dir.src.images}/libero-logo.svg`;
+  config.files.src.favicon = `${config.dir.src.images}/libero-logo-small.svg`;
```

Comment:
Wanted to use `libero-logo.svg#graphic`, but can't find any JS library that understands SVG fragments. 😞